05-Organizing-and-Executing-Simulations/02-Executing-Simulations.ipynb

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
<p>Switch to place <code>flow.FlowProject.operation</code> first. Also, create a subclass of <code>FlowProject</code> to use which is convention.</p><p> </p><p>Use <code>@flow.FlowProject.post.true("randomized")</code> ﻿(The same for the precondition).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I wasn't sure which path to take here as some signac examples make an empty subclass and some do not.

What verbiage would you suggest to explain the need for this to a new HOOMD/signac-flow user?

Reply:
<p>What do you mean by place "flow.FlowProject.operation" first? It is first in this code block.</p>